README.md

```diff
@@ -51,29 +86,66 @@ jobs:
 <!-- start configuration-options -->
 ```yaml
 - uses: nrwl/nx-set-shas@v1
+  env:
+    # The github token needed to provide access to APIs.
+    # Use ${{ secrets.GITHUB_TOKEN }} as a value
+    #
+    # Required: true
+    GITHUB_TOKEN: ''
```

Comment:
We should use `${{ github.token }}` rather than forcing the user to set up a secret in their repo and then reference that in the env.

One of the great things about github actions is the fact you get a token provisioned automatically for you

Reply:
See examples on the official org of actions: https://github.com/actions

Reply:
User does not need to set up new secret in their repo, the `GITHUB_TOKEN` exists by default. `GITHUB_XYZ` and `github.xyz` are the same. For some reason, it wasn't working when I tried to reference the env value in the `action.yml` without specifying it in the workflow. But works fine with GitHub context. Thanks!

https://github.com/octokit/action.js/blob/master/README.md